lib/config.go

```diff
@@ -34,14 +35,15 @@ type GetConfigParams struct {
 
 // GetConfig returns the Config, or one of the specified fields of the Config,
 // as a slice of bytes the bytes can be formatted as json, concise json, or yaml
-func (m *ConfigMethods) GetConfig(p *GetConfigParams, res *[]byte) (err error) {
-	if m.inst.rpc != nil {
-		return checkRPCError(m.inst.rpc.Call("ConfigMethods.GetConfig", p, res))
+func (m *ConfigMethods) GetConfig(ctx context.Context, p *GetConfigParams) ([]byte, error) {
+	if m.inst.http != nil {
+		return nil, fmt.Errorf("config is not supported across nodes")
```

Comment:
on second thought, this error is the one users will when they try to `qri config get` while running `qri connect` somewhere else. We should make this a standard error `ErrUnsupportedRPC` so we can capture it with `errors.Is` & present a clearer error for CLI users ("this could mean you're running qri connect in another terminal")